source/include/azure_iot_hub_client.h

```diff
@@ -21,7 +21,7 @@
 #include "azure_iot_mqtt_port.h"
 #include "azure_iot_transport_interface.h"
 
-#define hubclientSUBSCRIBE_FEATURE_COUNT    3
```

Comment:
How about creating separate small PR for header. So, that we have less conflict

Reply:
yea I was going to but these changes were simple enough. All the other ones I've created issues for.